firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -172,8 +146,100 @@ def execute_dry_run_import(import_job_instance_id, *args, **kwargs):
             )
             error_row_instance.save()
 
-    # Connecting back search indexing signals
-    # connect_signals()
-    # logger.info("Re-connected all search index related signals")
+    return report
+
+
+@shared_task
+def batch_import(import_job_instance_id, dry_run=True):
+    # This method passes the provided CSV file through the clean method,
```

Comment:
This method has a lot of logic that is different depending on the value of dry_run. Is it possible to split those out, so we have something more like this?

def batch_import(... dry_run=True):
  # common logic
  if(dry_run):
    awesome_dry_run_method(...)
  else:
    awesome_import_method(...)
 # common logic
 # the end

Reply:
I like this new version a lot. Thanks for making the changes!